From 94ea39d4645296e515ecc45ebb989c3f9d802c32 Mon Sep 17 00:00:00 2001
From: Andy Fleming <afleming@gmail.com>
Date: Wed, 16 Mar 2016 23:15:44 -0500
Subject: [PATCH 46/57] powerpc: Fix incorrect PPC32 PAMU dependency

The Freescale PAMU can be enabled on both 32 and 64-bit Power
chips. Commit 477ab7a19cec8409e4e2dd10e7348e4cac3c06e5
(iommu: Make more drivers depend on COMPILE_TEST)
restricted PAMU to PPC32. PPC covers both.

Signed-off-by: Andy Fleming <afleming@gmail.com>
---
 drivers/iommu/Kconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/iommu/Kconfig b/drivers/iommu/Kconfig
index b9094e9..5ccd46c 100644
--- a/drivers/iommu/Kconfig
+++ b/drivers/iommu/Kconfig
@@ -57,8 +57,7 @@ config IOMMU_DMA
 
 config FSL_PAMU
 	bool "Freescale IOMMU support"
-	depends on PPC32
-	depends on PPC_E500MC || COMPILE_TEST
+	depends on PPC_E500MC || (COMPILE_TEST && PPC)
 	select IOMMU_API
 	select GENERIC_ALLOCATOR
 	help
-- 
2.10.2

